examples/basic/axes/twin_axes.py

```diff
@@ -10,24 +10,27 @@
 from numpy import arange, linspace, pi, sin
 
 from bokeh.models import LinearAxis, Range1d
+from bokeh.palettes import Sunset6
 from bokeh.plotting import figure, show
 
 x = arange(-2*pi, 2*pi, 0.2)
 y = sin(x)
 y2 = linspace(0, 100, len(x))
 
+blue, red = Sunset6[2], Sunset6[5]
+
 p = figure(x_range=(-2*pi, 2*pi), y_range=(-1, 1))
 p.background_fill_color = "#fafafa"
 
-p.scatter(x, y, color="crimson", size=8)
-p.yaxis.axis_label = "red circles"
-p.yaxis.axis_label_text_color ="crimson"
+p.scatter(x, y, line_color="black", fill_color=blue, size=12)
+p.yaxis.axis_label = "light blue circles"
+p.yaxis.axis_label_text_color ="blue"
```

Comment:
Sorry, I could have been more specific with suggestion:
```suggestion
p.yaxis.axis_label_text_color =blue
```